core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -393,6 +394,7 @@ private static String toSql(RelNode root, SqlDialect dialect,
         .withMysql().ok(expectedMysql)
         .withClickHouse().ok(expectedClickHouse)
         .withPresto().ok(expectedPresto)
+        .withTrino().ok(expectedPresto)
```

Comment:
Use `expectedPresto` for verification directly. What if the result is consistent with presto?

Reply:
This is for more clear for test case, and maybe not nessary to change, thanks ~